plugins/outputs/sql/sql.go

```diff
@@ -213,6 +220,14 @@ func (p *SQL) generateCreateTable(metric telegraf.Metric) string {
 	return query
 }
 
+func (p *SQL) generateAddColumn(tablename, column, columnType string) string {
+	query := p.TableUpdateTemplate
+	query = strings.ReplaceAll(query, "{TABLE}", quoteIdent(tablename))
+	query = strings.ReplaceAll(query, "{COLUMN}", quoteIdent(column)+" "+columnType)
+
+	return query
+}
```

Comment:
Instead of performing multiple `strings.ReplaceAll` calls, `fmt.Sprintf` might be more readable and efficient.
```suggestion
func (p *SQL) generateAddColumn(tablename, column, columnType string) string {
	table := quoteIdent(tablename)
	col := quoteIdent(column)

	return fmt.Sprintf(p.TableUpdateTemplate, table, col+" "+columnType)
}

```

Reply:
Please leave it as it is, then. Thanks!